### PR TITLE
💣 fix(Icon): update toiletries name

### PR DESCRIPTION
**Pull request by diegomarcuz**
## Description 📄

This update the Toiletries icon name

## Platforms 📲

- [X] Web
- [X] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist: 🔍

- [ ] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [ ] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Screenshots 📸

[Upload your screenshots here]

| Before | After |
| ------ | ----- |
|        |       |

### CODE REVIEW

**Review by diogoverde — APPROVED** [review]

**Review by ericcleao — APPROVED** [review]

**Review by marinamsou — APPROVED** [review]

**Review by wellingtonnascimento — APPROVED** [review]

File: packages/icons/src/ui.js [hunk 1]
```diff
@@ -303,7 +303,7 @@ export {
   Time,
   TimeFilled,
   Token,
-  Tolietries,
```
Comment (caiotracera) [inline packages/icons/src/ui.js:306]: Is there a chance this change will break some application that is using this icon with the wrong name?

Comment (diegomarcuz) [reply packages/icons/src/ui.js:306]: good point that's why we are trying to address it as a breaking change

Comment (diegomarcuz) [reply packages/icons/src/ui.js:306]: what do you think?

Comment (caiotracera) [reply packages/icons/src/ui.js:306]: I like the approach of addressing this as a breaking change. In addition, I would also suggest to keep two exports: one with the wrong name (just like it is right now), and another one with your fix.

```jsx
export {
...
Tolietries,
Toiletries: Tolietries,
....
}
```

That way, we allow teams time to fix the problem before it happens. That way, we allow teams time to fix the problem before it happens. And therefore, we have set a deadline for the incorrect version to be removed, such as 30 days. WDYT?

CC @matheushdsbr @caioalexandrebr

**Pull request merged**